src/js/disability-benefits/components/ClaimEstimate.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import moment from 'moment';
+
+export default function ClaimEstimate({ maxDate }) {
+  const estimatedDate = moment(maxDate);
+  const today = moment().startOf('day');
+
+  if (!maxDate || !estimatedDate.isValid() || estimatedDate.isAfter(moment(today).add(2, 'years'))) {
```

Comment:
Also, `today` is already a moment object, do you need to wrap it again to add 2 years?